go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -411,6 +422,21 @@ func TestWithDefaultKeyspaceFromFileSharded(t *testing.T) {
 	testFile(t, "select_cases_with_default.txt", testOutputTempDir, vschema)
 }
 
+func TestWithUserDefaultKeyspaceFromFileSharded(t *testing.T) {
+	// We are testing this separately so we can set a default keyspace
+	vschema := &vschemaWrapper{
+		v: loadSchema(t, "schema_test.json", true),
+		keyspace: &vindexes.Keyspace{
+			Name:    "user",
+			Sharded: true,
+		},
+		tabletType: topodatapb.TabletType_PRIMARY,
+	}
+
+	testOutputTempDir := makeTestOutput(t)
+	testFile(t, "select_cases_with_user_as_default.txt", testOutputTempDir, vschema)
+}
```

Comment:
I might be missing, but I don't understand the clear difference between `TestOneWithUserAsDefault` and `TestWithUserDefaultKeyspaceFromFileSharded`. Is `select_cases_with_user_as_default.txt` a file where we will keep adding plan tests, or a file that serves for one-case test (like `onecase.txt`)?

Reply:
yes. `TestOne...` is for one-offs, and `TestWithUserDefaultKeyspaceFromFileSharded` is forever